iree/compiler/Dialect/Flow/Transforms/test/identify_dispatch_regions2_hlo.mlir

```diff
@@ -16,65 +16,40 @@ func @simpleMath(%arg0 : tensor<4xf32>) -> tensor<4xf32> {
 
 // -----
 
-// CHECK-LABEL: @hloElementwiseOps
-func @hloElementwiseOps(%arg0 : tensor<4xf32>) -> tensor<4xf32> {
-  // CHECK-NEXT: %[[WORKLOAD:.+]] = constant 4
-  // CHECK-NEXT: %0 = flow.dispatch.region
-  // CHECK-SAME: [%[[WORKLOAD]] : index]
-  // CHECK-SAME: (%arg1 = %arg0 : tensor<4xf32>) -> tensor<4xf32> {
-  // CHECK-NEXT:   %1 = xla_hlo.add %arg1, %arg1 : tensor<4xf32>
-  %0 = xla_hlo.add %arg0, %arg0 : tensor<4xf32>
```

Comment:
don't we still want this test to ensure these are put in the same region?

Reply:
See comment above.